### ISSUE TITLE
Discussion: Archive Feedstock?

### ISSUE THREAD

**Issue by mfansler** [conda-forge/jasp-feedstock#26]
This is a massive build that has sat stagnant for a year and doesn't seem to autopilot very well. Unless there is a maintainer willing to give it some TLC, I suggest it be archived.

@conda-forge/r @conda-forge/jasp thoughts?

**Comment by izahn** [comment]
Yes, it can be archived. Nothing depends on it so there will be no downstream problems.
